string class_name
float32 confidence
float32 distance_m
int32 x1
int32 y1
int32 x2
int32 y2

